config/schema/field_definitions.json

```diff
@@ -508,5 +508,10 @@
   "content_store_document_type": {
     "description": "The document type as stored in the Content Store",
     "type": "identifier"
+  },
+
+  "part_of_taxonomy": {
+    "description": "Any taxon tagged to the document and any of their ancestor taxons",
```

Comment:
Right of course. I confused myself, I think I was thinking of the way you retrieve the pages.